expertise/create_dataset.py

```diff
@@ -289,12 +290,23 @@ def retrieve_expertise(self):
 
     def get_submissions(self):
         invitation_ids = self.convert_to_list(self.config.get('paper_invitation', []))
+        paper_id = self.config.get('paper_id')
         submissions = []
 
         for invitation_id in invitation_ids:
             submissions.extend(list(openreview.tools.iterget_notes(
                 self.openreview_client, invitation=invitation_id)))
 
+        if paper_id:
+            submissions.append(self.openreview_client.get_note(paper_id))
+
+        # Bug: specter+mfr cannot handle a single submission
+        # Solution: create a copy of the note and modify the ID
+        if self.config.get('model') == 'specter+mfr' and len(submissions) == 1:
+            dummy = deepcopy(submissions[0])
```

Comment:
instead of adding a library for copying one single submissions, could you just create the new submission with the properties that we already know the submission has? id, title, abstract.